Automattic/jetpack
# Protect: Add brute force protection toggle

#### Description
Adds a brute force protection toggle to the WAF tab in the Protect UI that toggles the "protect" module and the Brute force protection feature.

## Proposed changes:

* Adds a Brute force toggle to the Protect and enables and sets up the module by default
* Adds Brute force functionality to Protect that is activated when the module is enabled

### Other information:

## Jetpack product discussion


- 1201069996155224-as-1203716242675366

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Start up a Jurassic Ninja site using this Protect branch
* Activate Protect and initiate a site-only connection
* Ensure that the Brute force module is enabled by default and that the toggle in the UI reflects the same
* Toggle the module off and verify that the UI responds accordingly
* Re-enable the setting and proceed to the site login in an incognito window
* Fail the login repetitively and verify that Brute force reacts as expected, first with a math auth prompt after several failed attempts and lastly with a hard block
* Ensure all other Brute force functionality remains intact and working
* Enable Jetpack and ensure that the two can work in tandem
* Verify that no regression in functionality is introduced
